<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Display</class>
 <widget class="QDialog" name="Display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ButtonDisplayReset">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelDisplay">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
